react/features/calendar-sync/components/MeetingList.native.js

```diff
@@ -73,16 +77,9 @@ class MeetingList extends Component<Props> {
      * @inheritdoc
      */
     componentWillReceiveProps(newProps) {
-        // This is a conditional logic to refresh the calendar entries (thus
-        // to request access to calendar) on component first receives a
-        // displayed=true prop - to avoid requesting calendar access on
-        // app start.
-        if (!this._initialLoaded
-                && newProps.displayed
-                && !this.props.displayed) {
+        if (newProps.displayed) {
```

Comment:
Does checking solely for newProps.displayed in componentWillReceiveProps, compared to value change, cause the action to get dispatched more often than expected?

Reply:
Well, Emil actually wants it to be even more aggressive... But you're right, I'll change this because this is a less controllable side effect here